db/migrate/20230117220510_add_foreign_key_to_like.rb

```diff
@@ -0,0 +1,6 @@
+class AddForeignKeyToLike < ActiveRecord::Migration[7.0]
+  def change
+    add_column :likes, :author_id, :integer
+    add_column :likes, :post_id, :integer
```

Comment:
- [ ] Kindly add referential integrity unlike the columns which you have added to the migration files.
example
![image](https://user-images.githubusercontent.com/81932478/213060201-87aa76c9-fce8-44d4-aa51-bf6af6525173.png)